packages/shared/src/components/icons/Search/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Icon, { Size } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+type Props = {
+  filled?: boolean;
+  size?: Size;
+};
```

Comment:
Tagging for visibility regarding the concern for shared interface.

Reply:
Done